scalability/run-p8s.sh

```diff
@@ -1,190 +0,0 @@
-#! /usr/bin/env nix-shell
```

Comment:
Can `scalability/run-p8s.sh` and `scalability/run-p8s.nix` be moved to `/rs/tests/` and owned by `@dfinity/idx`? Some folks are still using these scripts to look at p8s data from a passed system-test run.

Reply:
Sure, I'll move them

Reply:
I moved the scripts to `/rs/tests`. I see that that folder is already owned by the IDX team, so I didn't add new entries in the code owners file.

Reply:
@ulan you need an extra `dirname` [here](https://github.com/dfinity/ic/blob/9e9e7ed6640c58cbbeda0ea9927f69982bf5daa0/rs/tests/run-p8s.sh#L10)

Reply:
Thanks! I simplified the expression now that the script is in `/rs/tests`.